apps/opik-documentation/documentation/fern/docs/production/anonymizers.mdx

```diff
@@ -121,8 +121,8 @@ def anonymize(self, data, **kwargs):
 
 The `anonymize()` method also receives additional context through `**kwargs`:
 
-- **`field_name`**: Indicates which field is being anonymized (`"input"`, `"output"`, or `"metadata"`)
-- **`object_type`**: The type of the object being processed (typically `dict` for trace/span data)
+- **`field_name`**: Indicates which field is being anonymized (`"input"`, `"output"`, `"metadata"`, or nested field names in dots notation such as `"metadata.email"`)
+- **`object_type`**: The type of the object being processed (`opik.message_processing.messages.CreateSpanMessage`, `opik.message_processing.messages.CreateTraceMessage`, `opik.message_processing.messages.UpdateTraceMessage`, `opik.message_processing.messages.UpdateSpanMessage`, `opik.message_processing.messages.CreateSpansBatchMessage`, or `opik.message_processing.messages.CreateTracesBatchMessage`)
```

Comment:
1. These types are not part of our public API, and it's probably better to keep them internal.
2. I'm not sure that referencing batching here makes sense since, according to the code, we anonymize each batch item separately.

How about just passing literals or enums: "span" and "trace"? I think it should be enough to describe what kind of data is being processed, and we won't expose our internal details. Anyway, only the specific fields from these objects will be passed to the anonymizers, so there is no need to know any other information about the payload.